header.php

```diff
@@ -88,7 +88,8 @@
 			</div>
 			<div class="header__nav">
 				<a class="header__nav-icon js-header-nav-toggle" href="#navigation"><?php _e( 'Toggle Menu', 'pressbooks-book' ); ?><span class="header__nav-icon__icon"></span></a>
-				<nav class="js-header-nav" id="navigation">
+				<nav aria-labelledby="book-toc" class="js-header-nav" id="navigation">
+					<h2 id="book-toc" class="screen-reader-text">Book Contents Navigation</h2>
```

Comment:
I feel h2 is not the right tag, maybe an span for this purpose? what do you think?

Reply:
@arzola yes, I just looked and there's not an `<h1>` that appears before this tag so an `<h2>` wouldn't follow the proper heading order. I'd suggest a `<p>`.

Reply:
```suggestion
					<p id="book-toc" class="screen-reader-text">Book Contents Navigation</p>
```